src/app/utils/utils.js

```diff
@@ -1,3 +1,5 @@
+import { URL } from "@canonical/jaaslib/lib/urls";
```

Comment:
This doesn't look like it's used?

Reply:
I didn't explicity add that.. must have been VS Code - removing.